content/events/2018/tech-day-02-03.md

```diff
@@ -0,0 +1,17 @@
+---
+title: February Tech Day
+date: 2018-02-03
+---
+
+{{< location 25 >}}
```

Comment:
We use `{{% ... %}}` elsewhere (and in most places I've seen similar tags); is there a difference between those and this angle brackets spelling? If not, it would be good to be consistent.

I don't have a preference which way, though I guess I'm more used to `%` from jinja and django. That might be a push in either direction though.

Reply:
The difference mainly affects itself when the shortcode has some content, using `%` renders the inner content with the markdown parser, whilst `<>` doesn't. Apparently in this case, it doesn't make a difference, will change to `%`.

Reply:
Aha, interesting.